home.html

```diff
@@ -0,0 +1,152 @@
+</head>
+<body>
+    <head>
+        <div class="main-container">
+        <div class="mobile-nav">
+            <div class="moblile-menu-icon">
+                <!-- <a id="mobile-icon-btn" href="#"> <img src="./assets/images/mobile_menu.png" alt="menu hamburger icon"></a> -->
+            </div>
+            <div id="myNav" class="overlay">
+                <a href="javascript:void(0)" class="closebtn" onclick="closeNav()">&times;</a>
+                <div class="overlay-content">
+                    <a href="./About.html" onclick="closeNav()">About</a>
+                    <a href="#about-social" onclick="closeNav()">Program</a>
+                    <a href="#contact" onclick="closeNav()">Join</a>
+                    <a href="#News" onclick="closeNav()">News</a>
+                    <a href="#contact" onclick="closeNav()">CC Campaign</a>
+                </div>
+            </div>
+            <div class="desk-nav">
+                <div class="logo-png">
+                    <a href="#"> <img src="./assets/images/logopng.png" alt="logo image"> </a>
+                </div>
+                <div class="desk-nav-items">
+                    <ul>
+                        <li class="menu-item"><a href="./About.html"> About </a> </li>
+                        <li class="menu-item">Progam</li>
+                        <li class="menu-item">Join</li>
+                        <li class="menu-item">Sponsor</li>
+                        <li class="menu-item">News</li>
+                    </ul>
+                </div>
+            </div>
+            <nav class="flex-nav"><button onclick="openNav()">☰</button></nav>
+        </div>
+    </head>
+    <!-- coding the hero section here -->
+        <section class="hero-section">
+            <img id="main-img" src="./assets/images/main-image.png" alt="">
+            <div class="message-main">
+                <p>
+                    Experts from all around the globe, gathering to share their experiences, and to invest in your ideas, your dream is coming true, just prepare your pitch and impress one investor at least to get the fund
+                </p>
+            </div>
+            <div class="location-data">
+                <p id="date">2022.10.14 (Wed) ~ 17 (Sat)</p>
+                <p id="place">@ National Museum of Korea, Art Center Nabi, etc.</p>
+            </div>
+        </section>
+    <!-- coding the main program section here -->
+        <section class="main-program">
+            <h2>Main Program</h2>
+            <div class="program-card card1">
+                <div class="prog-icon">
+                    <img src="./assets/images/program_icon_01.png" alt="program-icon-1">
+                </div>
+                <div class="prog-title">
+                    confrence
+                </div>
+                <div class="prog-desc"> 
+                    Listen to lectures from speakers from around the world and learn about the latest trends in the world.
+                </div>
```

Comment:
- [ ] For better accessibility of your website, it is better not to make too much use of `<div>`s when you can actually use semantic HTML. For your project titles and descriptions please replace the `<div>`s with a `<h3>` for the title and/or a <p> tag for the description in my opinion 😄